app/storage/encrypted_questionnaire_database_storage.py

```diff
@@ -23,28 +22,29 @@ def generate_key(user_id, user_ik, pepper=settings.EQ_SERVER_SIDE_STORAGE_ENCRYP
     return to_bytes(cek)
 
 
-class EncryptedStorage(DatabaseStorage):
+class EncryptedQuestionnaireDatabaseStorage(QuestionnaireDatabaseStorage):
```

Comment:
I actually think the original class names were better. Could be QuestionnaireStorage and EncryptedQuestionnaireStorage and change QuestionnaireStore to something else?